src/styles.css

```diff
@@ -9,10 +9,10 @@
     --secondary-color:#FBDC57;
   }
 }
-
+/*
```

Comment:
Why comment these?

Reply:
@akshara-sun

Reply:
Will fix these!

Reply:
@amilajack I clicked on the "Resolve Conflicts" and took out the comments and "Marked As Resolved"

Reply:
Not sure what I need to do in Sound.js to fix the merge conflicts